golang/internal/pointer/pointer.go

```diff
@@ -0,0 +1,8 @@
+package pointer
+
+/*
+ * Creates a new pointer of type T
+ */
```

Comment:
Multiline comments are for package behavior description, functions generally have C++ style comments as per https://tip.golang.org/doc/comment

As simple as it is, this introduces untested code. Please add a simple unit test. This could also totally be fuzzed, what do you think?

Reply:
Would something like the function below be sufficient.  I can learn a little about fuzz testing and add it here.
```

func TestNewPTR(t *testing.T) {
	sPtr := NewPTR[string]("")
	assert.Equal(t, *sPtr, "")

	iPtr := NewPTR[int](120)
	assert.Equal(t, *iPtr, 120)
}

```

Reply:
Only if you are really interested doing so, otherwise the proposed test is fine.

Reply:
It should be fixed now.